### PR TITLE
Issue #5 add 2 fa testing

### PR DESCRIPTION
**Pull request by Aleksa-Vig**
Added acceptance tests for 2FA

TODO:
switch the URL for which selenium will run tests on from localhost to the hosted IP

### CODE REVIEW

File: tests/features/enable-2fa.feature [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Feature: Add tasks to todo list
+    As a user,
+    I want to be able to have Two factor protection on my account,
+    So that I feel safer about my account against potential threats
```
Comment (Brady-Malott) [inline tests/features/enable-2fa.feature:4]: Feature title needs to be changed

File: tests/features/steps/Enable-2fa-steps.py [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    options.add_argument("--disable-extensions")
+    context.driver = webdriver.Chrome(service=ChromeService(ChromeDriverManager().install()))
+    context.driver.implicitly_wait(5)
+    context.driver.get(BASETESTURL)
+
+    login(context.driver)
+    
+    context.driver.get(f'{BASETESTURL}/user/{USERNAME}')
+    context.driver.get(f"{BASETESTURL}/auth/enable_2fa")
+
```
Comment (Brady-Malott) [inline tests/features/steps/Enable-2fa-steps.py:38]: Why are we using two get requests here?

**Review by Brady-Malott — CHANGES_REQUESTED** [review]
See comments.

**Comment by Aleksa-Vig** [comment]
> See comments.

Addressed issues

**Review by Brady-Malott — APPROVED** [review]
LGTM

**Pull request merged**